config/initializers/setup_mail.rb

```diff
@@ -0,0 +1,12 @@
+if Rails.env.development?
+	ActionMailer::Base.delivery_method = :smtp
+	ActionMailer::Base.smtp_settings = {
+		address: 'smtp.sendgrid.net',
+		port: '587',
+		authentication: :plan,
+		user_name: ENV['SENDGRID_USERNAME'],
+		password: ENV['SENDGRID_PASSWORD'],
+		domain: 'heroku.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.